### PR TITLE
implement getting the definition of a node

### PR DESCRIPTION
**Pull request by dbu**

### CODE REVIEW

File: src/Jackalope/Item.php [hunk 1]
```diff
@@ -514,6 +517,43 @@ public function revert()
...
+        $fallbackDefinition = null;
+        $types = $this->getParent()->getMixinNodeTypes();
+        array_push($types, $this->getParent()->getPrimaryNodeType());
+        /** @var $nt NodeTypeInterface */
+        foreach ($types as $nt) {
+            /** @var $candidate ItemDefinitionInterface */
+            foreach ($definitions($nt) as $candidate) {
+                if ($candidate->getName() === $this->name) {
+                    return $candidate;
+                } elseif ('*' == $candidate->getName()) {
```
Comment (lsmith77) [inline src/Jackalope/Item.php]: ``if`` would be sufficient here

Comment (dbu) [inline src/Jackalope/Item.php]: fixed

**Comment by dbu** [comment]
@danrot does this help you? can you check if it works for you?

**Comment by lsmith77** [comment]
seems like something changed at styleci .. also do we have a PR for doctrine dbal or at least verified it doesn't cause issues there?

**Comment by danrot** [comment]
From a quick look I guess it should help... Will check as soon as possible, but might take a few days, just returned from holiday.

**Comment by dbu** [comment]
rebased and all green. i created a pull request on dbal now and updated the jackrabbit one.

**Pull request merged**